.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
```

Comment:
your configuration is ok.

please research more about the recommended eslint rules. e.g. what is the recommended one in the industry?

Reply:
"rules": {
        "react/prop-types": "off",
        "curly": "error",
        "eqeqeq": "off",
        "no-multi-spaces": "error",
        // "quotes": ["error", "double"]

    }

try some best practices rules. but removed the quotes one because some many '' need to fix.....